assets/src/components/tabBar.tsx

```diff
@@ -214,4 +215,13 @@ const showAppcue = (appcueId: string): void => {
   }
 }
 
+const readDispatcherFlag = (): boolean => {
+  const data = appData()
+  if (!data || data === undefined || data.dispatcherFlag === undefined) {
```

Comment:
the `data === undefined` is redundant because `!undefined` is `true`

Reply:
Now that you point it out, the `data.dispatcherFlag === undefined` is also redundant. I'll remove both.